Formula/joshua.rb

```diff
@@ -14,24 +14,24 @@ class Joshua < Formula
     sha256 "176fa47a6a2722fb5b6bf1e2efba8da32bab6355f3d844424a817882ed7b3a8e" => :mavericks
   end
 
-  option "with-es-en-phrase-pack", "Build with Spanish–English phrase-based model [1.9 GB]."
```

Comment:
The new language packs I've update with are compatible with stable and HEAD. Thanks for checking out.